<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="redLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="greenLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="blueLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="redEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="greenEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="blueEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="redSlider">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="greenSlider">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="blueSlider">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="colorView">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="randomButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QDial" name="redDial">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="greenDial">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="blueDial">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="redLabel_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="greenLabel_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="blueLabel_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>redSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sliderUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sliderUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sliderUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>editUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>editUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>editUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sliderUpdate()</slot>
  <slot>editUpdate()</slot>
  <slot>buttonClicked()</slot>
 </slots>
</ui>
